README.md

```diff
@@ -38,7 +40,7 @@ bundle install
    - ``export SECRET_KEY_BASE=`rake secret` ``
    - `rake db:setup`
    - `rake figgy:development`
-   - In a separate terminal: `rails s`
+   - In a separate terminal: `foreman start`
```

Comment:
I get `foreman: command not found` — is there more to add to the setup instructions?

Reply:
We added the foreman gem to the dev group in the Gemfile